docs/manual.md

```diff
@@ -609,14 +587,14 @@ The input file name and output file name are explicitly specified using the -v a
     200|LRH Server socket closed
 
 To convert from other file formats, for which a corresponding loader does exist, you should be able to adjust this project file.
+--> 
 
-<a name="advanced-usage"></a>
-
+<!-- ###################################################################### -->
 ## Advanced Usage
```

Comment:
I think, we should move this to a different document.

Reply:
No more used/required ...  devguide can be used ...